Libplanet/Store/Trie/IKeyValueStore.cs

```diff
@@ -13,5 +15,12 @@ public interface IKeyValueStore
         public void Delete(byte[] key);
 
         public bool Exists(byte[] key);
+
+        /// <summary>
+        /// Lists all keys that have ever had stored.
```

Comment:
Is this explanation accurate?  This docs — because of the expression “have ever had stored” — apparently is read to mean that if a key `k` has been `Set(k)` once, this `ListKeys()` must return (an enumerable yielding) `k` regardless of `Delete(k)` has been called or not.

Reply:
You're right, I will update it.

Reply:
If I explained it as *Lists all keys that have stored in the storage.*, is there no issues to misunderstand?

Reply:
That would be good if it's adjusted to “… have *been* stored …”.